### PR TITLE
[Doc Improvement] [Update - Developer preview app manifest][3661653]

### PR DESCRIPTION
**Pull request by mobajemu-msft**
* Backfilling existing dev preview and GA manifest reference docs with snippets of top-level elements that are missing.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b9b60ffc018d2d46228eee5042e6ac52765de808/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 104 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/012478d9-73a6-4ba4-bcf1-22b9ed8dfe2a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 1]
```diff
@@ -208,6 +208,21 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
         "id": "AAD App ID",
         "resource": "Resource URL for acquiring auth token for SSO"
     },
+    "graphConnector": {
+        "notificationUrl": "The URL where Graph-connector notifications for the application must be sent"
```
Comment (v-vanv) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:212]: "notificationUrl": "The URL where Graph connector notifications for the application must be sent"

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 2]
```diff
@@ -208,6 +208,21 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
...
+    },
+
+    "showLoadingIndicator": {
+        "default": "A true or flase value indicating whether or not show loading indicator when app/tab is loading"
```
Comment (v-vanv) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:216]:
```suggestion
        "default": "A true or false value indicating whether or not show loading indicator when app/tab is loading"
```

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 3]
```diff
@@ -208,6 +208,21 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
...
+    },
+    "isFullScreen": {
+        "default": "A true o false value indicating whether a personal app is rendered without a tab header-bar"
```
Comment (v-vanv) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:219]:
```suggestion
        "default": "A true or false value indicating whether a personal app is rendered without a tab header-bar"
```

**Review by v-vanv — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/40220569d159a5532fd4f15f946eb307ad6361d4/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 104 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aafb6484-7a20-4861-848e-3bcc0c23c8e8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8352d460324e50ca0034a4a37be05e528f5a9b39/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 104 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f95d2798-02cc-461f-9ed0-c892f4807509) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-vanv — DISMISSED** [review]

**Review by erikadoyle — CHANGES_REQUESTED** [review]
Instead of copying over the schema description as the value for these properties, I think it'd be much more useful to show customers plausible example values to simulate how they might actually write their app manifest if they were using these features. For example, for the `actions` section, you could use values similar to the ones in [actions sample manifest](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/m365-actions-preview/nodejs/appManifest/manifest.json#L63-L120).

Longer term, as a separate doc improvement effort, it'd be awesome to ensure the entire manifest sample has plausible example values, rather than descriptions from the schema.

**Comment by Wajeed-msft** [comment]
> Instead of copying over the schema description as the value for these properties, I think it'd be much more useful to show customers plausible example values to simulate how they might actually write their app manifest if they were using these features. For example, for the `actions` section, you could use values similar to the ones in [actions sample manifest](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/m365-actions-preview/nodejs/appManifest/manifest.json#L63-L120).
> 
> Longer term, as a separate doc improvement effort, it'd be awesome to ensure the entire manifest sample has plausible example values, rather than descriptions from the schema.

I agree, we should add examples instead of description.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]
> Instead of copying over the schema description as the value for these properties, I think it'd be much more useful to show customers plausible example values to simulate how they might actually write their app manifest if they were using these features. For example, for the `actions` section, you could use values similar to the ones in [actions sample manifest](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/m365-actions-preview/nodejs/appManifest/manifest.json#L63-L120).
> 
> Longer term, as a separate doc improvement effort, it'd be awesome to ensure the entire manifest sample has plausible example values, rather than descriptions from the schema.

Agreed. We should include example instead of field description.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/161343f6aa68f41160e808e4ed0af07a3e3fdea6/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 103 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5fce0a83-26ff-4bf3-a201-d85a07f036f3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9b535453969112dd1bce16a9ab38225725c12ff2/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 103 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c3e6639-8550-4947-990c-a69414236b6a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 4]
```diff
@@ -208,6 +208,15 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
         "id": "AAD App ID",
         "resource": "Resource URL for acquiring auth token for SSO"
     },
+    "graphConnector": "https://contoso.com/connector/notification",
+    "showLoadingIndicator": false,
+    "isFullScreen": false,
+    "defaultBlockUntilAdminAction": false,
+    "publisherDocsUrl": "https://contoso.com/teamtabapp/admin-doc",
+    "scopeConstraints": {
```
Comment (mobajemu-msft) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:216]: @erikadoyle I could find no example snippet or use cases of scopeConstraints in the samples folder or in the docs, any guidance on this portion?

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 5]
```diff
@@ -919,6 +969,69 @@ The `extensions.runtimes` property configures the sets of runtimes and actions t
 
 To use `extensions.runtimes`, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command).
 
+```json
```
Comment (mobajemu-msft) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:972]: @erikadoyle this is a draft idea, and not a final representation but the section for the extensions is too lengthy to be placed in the snippet. It makes an already length doc near unbearable to scroll through. The option to split up that section and place it in the body is also not an apt choice, especially after viewing the learn preview site.  The optimal solution would likely be to link to its sample in the [Office Add-ins with the unified app manifest for Microsoft 365]( https://learn.microsoft.com/en-us/office/dev/add-ins/develop/unified-manifest-overview#see-also) doc. Thoughts?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/767be38082812dcc0c467505536d192e58ad9dcf/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 102 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/672b9947-1c99-4215-b904-2ea4c7aed3e8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a065c20a9dcdf1b4a33e0dad12b0ca9ca7611dd3/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 103 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3addad9f-e4af-49cf-a7ed-c7ea3e945dff) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md (hunk 4 above)
Comment (erikadoyle) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md:216]: Just messaged you with ideas to look for more info--

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b0ec04e850878b7d91c46dfc1feb016c0feedc33/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 103 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/61b77b99-23a7-444e-b1d7-a7d396dd3490) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-vanv — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3965d37e69516e86da2b0230872d2668e23d9574/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 103 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de1fcbe8-66c3-44c3-aa35-6545137f222a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 6]
```diff
@@ -208,6 +208,19 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
         "id": "AAD App ID",
         "resource": "Resource URL for acquiring auth token for SSO"
     },
+    "graphConnector": "https://contoso.com/connector/notification",
```
Comment (erikadoyle) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:211]: I don't think this is correct; the [reference says](https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema-dev-preview#graphconnector) that the `notificationUrl` child property is required for graphConnector. Per this [GH issue / feature request](https://github.com/MicrosoftDocs/msteams-docs/issues/6065), a sample isn't yet available for using this property, so I think we'll need to omit it here for now.

Comment (v-sdhakshina) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md:211]: @mobajemu-msft request you to confirm this.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/540b767c2a203f12d0b0b8c24da70904d98c4e4d/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 103 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0a5402c-4427-4eb5-a17c-4191b1f17eae) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md (hunk 6 above)
Comment (v-sdhakshina) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md]: Removed `notificationUrl` sub property as per the suggestion.

**Review by erikadoyle — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-sdhakshina — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e1a9ea2382b07d48cdc6a11e641e6c3c04a8d026/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 83 | 130 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22397954-2cd2-4eb1-a388-9bac790ed8cc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**